modules/reindex/src/main/java/org/elasticsearch/index/reindex/BulkByScrollTask.java

```diff
@@ -208,7 +215,8 @@ public XContentBuilder innerXContent(XContentBuilder builder, Params params)
             builder.field("batches", batches);
             builder.field("version_conflicts", versionConflicts);
             builder.field("noops", noops);
-            builder.field("retries", retries);
+            builder.field("bulk_retries", bulkRetries);
+            builder.field("search_retries", searchRetries);
```

Comment:
What about :
```json
"retries": {
  "bulk": 0,
  "search": 0,
}
```

Note: I tend to like JSON inner objects since clients and parsers can skip whole objects while parsing...